radixdlt/src/main/java/com/radixdlt/network/messaging/MessageCentralImpl.java

```diff
@@ -76,11 +75,11 @@
 	private final RateLimiter outboundLogRateLimiter = RateLimiter.create(1.0);
 
 	// Inbound message handling
-	private final BlockingQueue<MessageEvent> inboundQueue;
+	private final SimpleBlockingQueue<MessageEvent> inboundQueue;
 	private final SimpleThreadPool<MessageEvent> inboundThreadPool;
 
 	// Outbound message handling
-	private final BlockingQueue<MessageEvent> outboundQueue;
+	private final SimpleBlockingQueue<MessageEvent> outboundQueue;
```

Comment:
It's not evident to me why this was changed, could you elaborate?

Reply:
Primarily because this was previously a `PriorityBlockingQueue`.  Unfortunately the Javadoc for that class states "_Operations on this class make no guarantees about the ordering of elements with equal priority_", which completely breaks the LIFO behaviour we need to guarantee consensus messages are sent and received in order.